docs/reference/modules/uploadfs.md

```diff
@@ -183,7 +183,7 @@ The `key` option is used to pass the value of the `accessKeyId` to the `AWS.Cred
 The `token` option is used to pass the value of the optional `sessionToken` to the `AWS.Credentials()` credentials object. See the [official documentation](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#constructor_details) for alternative ways to present the credentials.
 
 ### `bucketObjectsACL`
-The `bucketObjectsACL` option sets the access control level for files uploaded to AWS S3, with a default of `public-read`. Setting this option to `private` ensures that only authorized users can access the files, enhancing security for sensitive content. This is useful when integrating with a CDN, where you can use signed URLs to securely serve private content to users.
+The `bucketObjectsACL` option sets the access control level (ACL) for files uploaded to AWS S3, with a default of `public-read`. If you want to make your S3 bucket private and serve content through the Amazon CloudFront service, you need to set `bucketObjectsACL: 'private'` to allow CloudFront access. Additionally, follow the [documentation](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-restricting-access-to-s3.html) to ensure your bucket is set up with an Origin Access Control correctly, otherwise, CloudFront will not be able to access it.
```

Comment:
"set `bucketObjectsACL: 'private'` to allow CloudFront access"

But it doesn't do that, it blocks access for everyone, and then the Origin Access Control grants it for CloudFront. Just want to avoid confusing wording.

Reply:
Changed